website/cue/reference/components/sinks/socket.cue

```diff
@@ -13,7 +13,7 @@ components: sinks: socket: {
 	}
 
 	features: {
-		acknowledgements: false
+		acknowledgements: true
```

Comment:
Need to update for `statsd` as well, right?

Reply:
No, `statsd` has `acknowledgements: sinks.socket.features.acknowledgements` so it copies the change from here.

Reply:
Ah, gotcha 👍 - didn't actually check the cue.